meta-lmp-base/conf/distro/lmp-wayland.conf

```diff
@@ -0,0 +1,13 @@
+require conf/distro/lmp.conf
+
+DISTRO = "lmp-wayland"
+DISTROOVERRIDES = "lmp:lmp-wayland"
+DISTRO_NAME = "Linux-microPlatform Wayland"
+
+DISTRO_FEATURES_append = " wayland opengl"
+
+#IMX8
+PREFERRED_VERSION_weston_mx8mm = "9.0.0"
```

Comment:
We can actually move these 2 changes into meta-lmp-bsp/conf/machine/include/lmp-machine-custom.inc under the imx8mm section.

Reply:
Ok will do.